memory/src/main/java/org/opensearch/ml/memory/action/conversation/CreateConversationTransportAction.java

```diff
@@ -86,7 +86,7 @@ protected void doExecute(Task task, CreateConversationRequest request, ActionLis
         try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().newStoredContext(true)) {
             ActionListener<CreateConversationResponse> internalListener = ActionListener.runBefore(actionListener, () -> context.restore());
             ActionListener<String> al = ActionListener.wrap(r -> { internalListener.onResponse(new CreateConversationResponse(r)); }, e -> {
-                log.error("Failed to create new conversation with name " + request.getName(), e);
+                log.error("Failed to create new memory with name " + request.getName(), e);
```

Comment:
please use parameterized logging in Java.
`"Failed to create new memory with name {}",request.getName()`